<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionContent</class>
 <widget class="QWidget" name="ConnectionContent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="brokerLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>200</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>tcp://locahost:1883</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disconnectButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>180</width>
     <height>240</height>
    </rect>
   </property>
   <property name="title">
    <string>Publish</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="publishButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>170</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Publish</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="publishTopicEdit">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>50</y>
      <width>160</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="publishPayloadEdit">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>120</y>
      <width>160</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="publishTopicLabel">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Topic:</string>
    </property>
   </widget>
   <widget class="QLabel" name="publishPayloadLabel">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>100</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Payload:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>270</width>
     <height>240</height>
    </rect>
   </property>
   <property name="title">
    <string>Subscribe</string>
   </property>
   <widget class="QLineEdit" name="subscribeTopicEdit">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>50</y>
      <width>160</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="subscribeTopicLabel">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Topic:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="subscribeButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Subscribe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="unsubscribeButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>85</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Unsubscribe</string>
    </property>
   </widget>
   <widget class="QComboBox" name="subscriptionsCombo">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>85</y>
      <width>160</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="subscriptionMessages">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>120</y>
      <width>260</width>
      <height>120</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
